apps/todaysname/todays_name.star

```diff
@@ -0,0 +1,88 @@
+"""
+Applet: Swedish Today's Name
+Summary: Shows today's name in Sweden
+Description: The app shows today's nameday names in Sweden.
+Author: y34752
+"""
+
+load("encoding/json.star", "json")
+load("render.star", "render")
+load("schema.star", "schema")
+load("re.star", "re")
+load("http.star", "http")
+load("cache.star", "cache")
+
+def getlistasstring(listin):
```

Comment:
Here's a common pattern for doing this type of thing: `"\n".join(listin)`. Not a blocker, but thought it might come in handy at some point! =)